functions/README_budspencer.md

```diff
@@ -0,0 +1,291 @@
+# Budspencer theme

```

Comment:
@ngmoviedo What's the purpose of this?

Reply:
I am not a fan of further cluttering my `functions` directory. For Fisher, I'd suggest removing this feature completely.